plugins/dialog/plugin.js

```diff
@@ -3440,6 +3478,31 @@ CKEDITOR.plugins.add( 'dialog', {
  * @member CKEDITOR.config
  */
 
+/**
+ * Allows to customize default dialog field values.
+ *
+ * Each field is represented by a flat object separated by `.` into three values used to
+ * identify it. E.g. `link.info.protocol` stands for:
+ *
+ * * `link` - dialog window name for which we want to customize dialog field value.
+ * * `info` - dialog window tab name in which the field is located.
+ * * `protocol` - the given field ID we want to customize.
+ *
+ * To make it easier for you to track field paths you may want to use
+ * [Developer Tools](https://ckeditor.com/cke4/addon/devtools) plugin.
+ *
+ * ```javascript
+ *  config.dialog_defaultValues = {
+ *    'link.info.protocol': 'https://',
+ *    'table.info.txtWidth': '100%'
+ *  }
+ * ```
+ *
+ * @since 4.11.0
+ * @cfg {Object} [dialog_defaultValues]
```

Comment:
We agreed some time ago on using more detailed Object type notation. Here You'd want to say what type keys and values are.

At this point we support only `String:String` (custom types gonna be supported as a value in followup PRs). So that's like `{Object.<String, String>}`.